src/commands/updateTaskbar.ts

```diff
@@ -16,17 +16,19 @@ export async function updateTaskbar() {
     return;
   }
 
-  const doc = activeEditor.document;
-  const filePath = doc.uri.fsPath;
-  const rootFolder = await getRootFolderPath();
-  if (!filePath.startsWith(rootFolder)) {
+  const filePath = activeEditor.document.uri;
+  const { guid, rootFolder } = await splitUri(filePath);
+
+  // in comment mode. still in the same file, just not in editor itself
+  if(guid === "mozilla.assay"){
```

Comment:
Spacing. (Weird, I'm surprised this wasn't auto-fixed by the linter.) 

```suggestion
  if (guid === "mozilla.assay") {
```